packages/regl-worldview/src/commands/Lines.js

```diff
@@ -389,7 +389,8 @@ const lines = (regl: any) => {
 
     const joined = primitive === "line strip";
     const effectiveNumPoints = numInputPoints + (shouldClose ? 1 : 0);
-    const instances = joined ? effectiveNumPoints - 1 : Math.floor(effectiveNumPoints / 2);
+    const guardedNumber = effectiveNumPoints > 0 ? effectiveNumPoints - 1 : 0;
```

Comment:
maybe call it `effectiveJoinedInstances` to be more explicit?

Reply:
Good catch! Can you simply use (after line 360)
```js
if (numInputPoints > 0) {
  return;
}
```
?

Reply:
Yeap it makes sense to early return.